__tests__/get-test.js

```diff
@@ -0,0 +1,51 @@
+import { get } from "../index.js";
```

Comment:
I moved and renamed these tests, not sure why it thinks it was a delete and new instead of a `git mv`